frontend/js/restqueries.js

```diff
@@ -0,0 +1,82 @@
+try {
+	var staticURL = "http://ec2-34-208-19-115.us-west-2.compute.amazonaws.com:18080/character/";
+	var playerURL = "OtherTestChar";
+	var DnDObj = getUser();
+	//getJSONField("player");
+}
+catch (err) {
+	console.log(err);
+}
+//getJSONField("name");
+populateDNDFields();
+
+function getUserText() {
+	var xhttp = new XMLHttpRequest();
+	xhttp.open("GET", staticURL + playerURL, false); //find a way to set async to true and have it return when it has something
+	xhttp.setRequestHeader("Content-type", "text/plain");
+	xhttp.send();
+	var response = JSON.stringify(xhttp.responseText);
+	console.log(response + " GetUserText() Response");
+	return response;
+}
+
+function getUser() {
```

Comment:
A few things here:

1. Function name isn't descriptive, since it's actually grabbing character details and not player details
2. Try not to use getBlah as function names, since that naming scheme is usually used for setters and getters. Something along the lines of "obtainCharacterDetails()" might work better.
3. This function should take in parameters instead of using global variables, probably along the lines of "obtainCharacterDetails(dbUrl,character)"